terraform/modules/rds/database.tf

```diff
@@ -68,10 +67,3 @@ resource "aws_db_instance" "replica" {
     prevent_destroy = true
   }
 }
-
-resource "aws_ssm_parameter" "database_connection_string" {
```

Comment:
Thought it was clearer to move this into a separate `parameters.tf` file (see `terraform/modules/rds/parameters.tf`) while adapting it